api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -457,6 +478,12 @@ private void verifyInvitationKey(String invitationKey){
       }
     }
 
+    user.setInstitutionalAffiliationSet(
```

Comment:
Does this take care of deleting the old entries when you do an update? (I always forget how this works.)

To avoid unnecessary deletes / inserts, you might want to do a comparison against existing institutional affiliation (ignoring IDs) to make sure something has changed before doing this.

Reply:
I checked that and confirmed that it does delete the old entries. I can add an equivalence check before updating.